table_pose_first:
  position: [0.869215607643, -5.00192499161, 0.0]
  orientation: [0.0, 0.0, -0.63459254249, 0.772846883293]

table_pose_second:
  position: [0.869215607643, -5.00192499161, 0.0]
  orientation: [0.0, 0.0, -0.63459254249, 0.772846883293]

table_order:
  position: [0.869215607643, -5.00192499161, 0.0]
  orientation: [0.0, 0.0, -0.63459254249, 0.772846883293]

init_pose_robot:
  position: [0.869215607643, -5.00192499161, 0.0]
  orientation: [0.0, 0.0, -0.63459254249, 0.772846883293]